go.mod

```diff
@@ -51,8 +51,8 @@ require (
 	github.com/google/go-github v17.0.0+incompatible // indirect
 	github.com/google/gofuzz v0.0.0-20170612174753-24818f796faf
 	github.com/gotestyourself/gotestyourself v2.2.0+incompatible // indirect
-	github.com/hashicorp/consul/api v0.0.0-00010101000000-000000000000
-	github.com/hashicorp/consul/internal v0.0.0-00010101000000-000000000000
+	github.com/hashicorp/consul/api v1.0.0
```

Comment:
Should we tie this to our product version?

Reply:
@pearkes long discussion previously with Mitchell/Paul, but -- no is the answer. :-)